app/uk/gov/hmrc/leakdetection/scanner/ExemptionChecker.scala

```diff
@@ -30,9 +30,17 @@ class ExemptionChecker @Inject()() {
       .filterNot(exemption =>
         excludedResults.exists(exclusion =>
           exemption.ruleId == exclusion.ruleId &&
-            exemption.filePath == exclusion.filePath &&
+            removeLeadingSlash(exemption.filePath) == removeLeadingSlash(exclusion.filePath) &&
             exemption.text.fold(true)(exclusion.lineText.contains)
         )
       )
   }
-}
+
+  private def removeLeadingSlash(filePath: String) = {
```

Comment:
should we call the function `normalise` - stripping of leading slashes being just one of the transformations?
(the extra `{}` are unnecessary)

Reply:
Done